src/utilcode/longfilepathwrappers.cpp

```diff
@@ -1288,7 +1287,7 @@ BOOL LongFile::IsPathNotFullyQualified(SString & path)
         return !IsDirectorySeparator(path[1]); // There is no valid way to specify a relative path with two initial slashes
     }
 
-    return !((path.GetCount() >= 3)           //The only way to specify a fixed path that doesn't begin with two slashes is the drive, colon, slash format- i.e. C:\
+    return !((path.GetCount() >= 3)           //The only way to specify a fixed path that doesn't begin with two slashes is the drive, colon, slash format- i.e. "C:\"
             && (path[1] == VolumeSeparatorChar)
```

Comment:
This line was previously commented out by accident it seems as the `\` of the `C:\` caused a line continuation of the comment, hence commenting the following line. The fix is simply to add parenthesis so the `\` character isn't the last.